### DISCUSSION TITLE
[FeatureRequest] Just output the snipet to command line but not run it

### DISCUSSION THREAD

**Discussion by letitfly** [knqyf263/pet#246] [Ideas]
Often I have to edit the command a little bit before run it.

I know I can do a "pet search" but I have to copy paste it on the command line again which is a little annoyance.

**Comment by hcvst** [comment]
I was looking for that too and added this function to my `.zshrc`,  @letitfly .

```
pets() 
  print -z $(pet search $*)
}
```

**Comment by RamiAwar** [comment]
I've also seen in some other places comments on how it's annoying to not have the command in local history, good idea to look into this at some point
